doc/classes/PhysicsServer3D.xml

```diff
@@ -608,7 +611,7 @@
 			<param index="0" name="body" type="RID" />
 			<param index="1" name="enable" type="bool" />
 			<description>
-				Sets whether a body uses a callback function to calculate its own physics (see [method body_set_force_integration_callback]).
+				Sets whether the body omits the standard force integration. If [param enable] is [code]true[/code], the body will not automatically use applied forces, torques, and damping to update the body's linear and angular velocity. In this case, [method body_set_force_integration_callback] can be used to manually update the linear and angular velocity instead.
```

Comment:
(Apparently I had an old pending review comment here that I hadn't actually submitted.)

The documentation seems to refer to this "standard force integration" in different ways. [`PhysicsDirectBodyState3D.integrate_forces`](https://docs.godotengine.org/en/stable/classes/class_physicsdirectbodystate3d.html#class-physicsdirectbodystate3d-method-integrate-forces) refers to it as the "built-in force integration code" and [`RigidBody3D.custom_integrator`](https://docs.godotengine.org/en/stable/classes/class_rigidbody3d.html#class-rigidbody3d-property-custom-integrator) refers to it as the "internal force integration". It might be a good idea to normalize all that, or at least pick one of the two existing ones.

It might also be a good idea to mention the `PhysicsDirectBodyState3D.integrate_forces` method somewhere here, and somewhere in `RigidBody3D.custom_integrator` as well, seeing as how that method is really only relevant when this is set to `true`, yet the two are never "connected" anywhere in the documentation as far as I can tell.